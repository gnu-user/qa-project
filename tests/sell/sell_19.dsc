TITLE
    Ticket Volume Input (valid input)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the sell 
    transaction. Enter a valid title, and enter valid numerical input (a double 
    with two decimal places between 0.00 and 999.99 inclusive) when prompted to 
    enter a ticket price. Enter an integer value between 0 and 100 inclusive 
    when prompted to enter a ticket volume. Verify that the value is accepted 
    without error.
DEPENDENCIES
    sell_15
