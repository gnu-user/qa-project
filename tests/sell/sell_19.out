Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the event title: 
Enter the sale price in dollars: 
Enter the number of tickets for sale: 
Tickets listed for sale successfully.